### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by Districh-ru**
Translate the recently added and changed texts.

### CODE REVIEW

File: files/lang/ru.po [hunk 1]
```diff
@@ -7303,23 +7318,20 @@ msgstr ""
...
 msgstr ""
 "%{skill} даёт %{count} процентов возможности выучить заклинание 1-го или 2-"
 "го круга, применённое в бою противником."
 
-#, fuzzy
 msgid ""
 "%{skill} gives your hero a %{count} percent chance to learn any given 3rd "
 "level spell (or below) that was cast by an enemy during combat."
 msgstr ""
 "%{skill} даёт %{count} процентов возможности выучить заклинание 1-го , 2-го "
```
Comment (ihhub) [inline files/lang/ru.po:7332]: I think a whitespace is not needed after **1-го** word.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Districh-ru , I left one comment here. Could you please check once you have time?

File: files/lang/ru.po (hunk 1 above)
Comment (Districh-ru) [reply files/lang/ru.po]: Hi, @ihhub, yes, thanks!

**Review by ihhub — APPROVED** [review]

**Review by Branikolog — APPROVED** [review]

**Comment by ihhub** [comment]
@Districh-ru , huge thanks for this update!

**Pull request merged**